measurePower.py

```diff
@@ -7,22 +7,24 @@
 import sys
 import yaml
 import json
-import jc
 import re
 import time
 
 class MeasurePower(Task):
-    def __init__(self, tc: TestConfig, node_name: str, tenant: bool):
+    def __init__(self, tc: TestConfig, node_name: str, tenant: bool, is_client: bool = False):
```

Comment:
Same comments as I had with measure CPU.

Reply:
Removed in recent update